### ISSUE TITLE
Invalid JSON log message 'decodeMsg: decoded message'

### ISSUE THREAD

**Issue by jakubgs** [status-im/nim-beacon-chain#3336]
Logstash fails to pars a `decodeMsg: decoded message` JSON log message with:
```
[WARN ][logstash.filters.json    ] Error parsing json {:source=>"raw", :raw=>"{\"lvl\":\"DBG\",\"ts\":\"2022-01-28 14:47:32.057+00:00\",\"msg\":\"decodeMsg: decoded message\",\"topics\":\"libp2p identify\",\"pubkey\":\"secp25...de929)\",\"addresses\":\"@[/ip4/127.0.1.1/tcp/9000/p2p/16Uiu2HAmDY7xcMg96tELqdUpqHuj1X72yJk23TgfpHyojhYbqUkG]\",\"protocols\":\"@[\"/ipfs/ping/1.0.0\", \"/meshsub/1.1.0\", \"/meshsub/1.0.0\"]\",\"observable_address\":\"Some(/ip4/95.217.87.121/tcp/50762)\",\"proto_version\":\"Some(\"ipfs/0.1.0\")\",\"agent_version\":\"Some(\"teku/teku/v22.1.1/linux-x86_64/-ubuntu-openjdk64bitservervm-java-11\")\"}", :exception=>#<LogStash::Json::ParserError: Unexpected character ('/' (code 47)): maybe a (non-standard) comment? (not recognized as one since Feature 'ALLOW_COMMENTS' not enabled for parser)
```
Full message
```
{"lvl":"DBG","ts":"2022-01-28 14:46:33.345+00:00","msg":"decodeMsg: decoded message","topics":"libp2p identify","pubkey":"secp25...86e5f)","addresses":"@[/ip4/144.91.91.97/tcp/13000]","protocols":"@["/ipfs/id/1.0.0", "/ipfs/id/push/1.0.0", "/meshsub/1.1.0", "/meshsub/1.0.0", "/floodsub/1.0.0", "/eth2/beacon_chain/req/status/1/ssz_snappy", "/eth2/beacon_chain/req/goodbye/1/ssz_snappy", "/eth2/beacon_chain/req/ping/1/ssz_snappy", "/eth2/beacon_chain/req/beacon_blocks_by_range/2/ssz_snappy", "/eth2/beacon_chain/req/beacon_blocks_by_root/2/ssz_snappy", "/eth2/beacon_chain/req/metadata/2/ssz_snappy"]","observable_address":"Some(/ip4/95.217.87.121/tcp/52794)","proto_version":"Some("ipfs/0.1.0")","agent_version":"Some("Prysm/v2.0.5/d6338f60427f471e278d23eb4098aa74f32a7a52")"}
```
Result of passing it to `jq`:
```
parse error: Invalid numeric literal at line 1, column 215
```

**Comment by arnetheduck** [comment]
cc @Menduist

**Comment by Menduist** [comment]
Is there a systematic fix for the Options, beside doing it manually? (`agent_version=if x.isSome(): $x.get() else: "None"`)

**Comment by arnetheduck** [comment]
@zah usually has some magic trick in his big black box :)

**Comment by Menduist** [comment]
Oh, found it `import json_serialization/std/options` :)

**Comment by jakubgs** [comment]
This appears to be fixed:
```
admin@metal-02.he-eu-hel1.nimbus.prater:~ % grep 'decodeMsg: decoded message' /var/log/service/beacon-node-prater-unstable-01/service.log | tail -n1 | jq 
{
  "lvl": "DBG",
  "ts": "2022-02-17 10:17:03.343+00:00",
  "msg": "decodeMsg: decoded message",
  "topics": "libp2p identify",
  "pubkey": "secp25...4183a)",
  "addresses": "@[/ip4/0.0.0.0/tcp/9000]",
  "protocols": "@[\"/ipfs/id/1.0.0\", \"/meshsub/1.1.0\", \"/eth2/beacon_chain/req/status/1/ssz_snappy\", \"/eth2/beacon_chain/req/ping/1/ssz_snappy\", \"/eth2/beacon_chain/req/metadata/1/ssz_snappy\", \"/eth2/beacon_chain/req/metadata/2/ssz_snappy\", \"/eth2/beacon_chain/req/beacon_blocks_by_range/1/ssz_snappy\", \"/eth2/beacon_chain/req/beacon_blocks_by_root/1/ssz_snappy\", \"/eth2/beacon_chain/req/beacon_blocks_by_range/2/ssz_snappy\", \"/eth2/beacon_chain/req/beacon_blocks_by_root/2/ssz_snappy\", \"/eth2/beacon_chain/req/goodbye/1/ssz_snappy\"]",
  "observable_address": "Some(/ip4/65.108.5.45/tcp/38742)",
  "proto_version": "Some(\"ipfs/0.1.0\")",
  "agent_version": "Some(\"nimbus\")"
}
```
Although not sure if it wouldn't make more sense to have these fields as just a normal list.
It's already a JSON, why not serialize the list to an actual list instead of a list printed as a string?

What do you think @Menduist?

**Comment by Menduist** [comment]
Weird, the fix was not merged into nimbus, so it should still not work

I'll merge it this week, it will switch to something like:
`"protocols": "/ipfs/id/1.0.0, /meshsub/1.1.0, etc"`
It's not ideal but we're a bit stuck because of nim standing issues

**Comment by jakubgs** [comment]
Possibly something else must have fixed the "stringification" of these fields, as before it was just invalid JSON.

**Comment by Menduist** [comment]
Should be fixed since v22.3.0, please reopen if it's still an issue
